### DISCUSSION TITLE
how to invoke `-print-tasty` and `-decompile`

### DISCUSSION THREAD

**Discussion by bishabosha** [VirtusLab/scala-cli#1984] [General]
the `scalac` command post 3.0.0 has options `-print-tasty` and `-decompile`. Are these meant to be subsumed by Scala CLI, or should the user continue to use `scalac`?

**Comment by lwronski** [comment]
Hi @bishabosha, I don't think there are currently any options to use it from Scala CLI, as we don't support `.tasty` files as inputs. It only works for me with `scalac`:

```
➜  scala-demo scalac .scala-build/project_585b944e27-25fccf1679/classes/main/Hello.tasty -print-tasty
--------------------------------------------------------------------------------
.scala-build/project_585b944e27-25fccf1679/classes/main/Hello.tasty
--------------------------------------------------------------------------------
Names:
   0: ASTs
   1: <empty>
   2: Hello
   3: Hello[ModuleClass]
   4: <init>
   5: <init>[Signed Signature(List(),Hello$) @<init>]
```

I think, it's a bug in Scala CLI and should be fixed.
